frontend/package.json

```diff
@@ -36,6 +36,8 @@
     "vue-property-decorator": "^9.1.2",
     "vue-router": "^3.2.0",
     "vue2-leaflet": "^2.7.1",
+    "vuelidate": "^0.7.7",
```

Comment:
why not https://v2.vuejs.org/v2/cookbook/form-validation.html ?

Reply:
@blueur I needed to have the $dirty logic. And I'm familiar with vuelidate.